helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/ZkClient.java

```diff
@@ -721,6 +721,9 @@ private void fireAllEvents() {
       });
       record(path, null, startT, ZkClientMonitor.AccessType.READ);
       return children;
+    } catch (ZkNoNodeException e){
```

Comment:
Run the template formatter to add a space? It is minor. But you know, it is a project style.

Reply:
Thanks. I fixed it.